### PR TITLE
Missing mailer

### PR DESCRIPTION
**Pull request by pg20**

### CODE REVIEW

File: app/models/email_template.rb [hunk 1]
```diff
@@ -42,6 +40,10 @@ def users
       User.mentee.where.not(registration_token: nil)
     when :unregistered_mentors
       User.mentor.where.not(registration_token: nil)
+    when :missing_mentors
+      User.mentor.where(is_missing: true).where(missing_since: ((Time.now - 7.days)..Time.now))
+    when :missing_mentees
+      User.mentee.where(is_missing: true).where(missing_since: ((Time.now - 7.days)..Time.now))
```
Comment (pg20) [inline app/models/email_template.rb]: Hi @DMips .. the missing mails will be sent every week ? Or the time when someone reported his/her partner missing ? 
the query will depend accordingly. Right now it send to mail to users missing in last 7days

File: app/models/email_template.rb [hunk 2]
```diff
@@ -4,17 +4,15 @@ class EmailTemplate < ActiveRecord::Base
...
+    missing_mentors: 4,
+    missing_mentees: 5,
     passed_mentees: 8,
     failed_mentees: 9,
     mentors_on_waiting_list: 10,
     rejected_mentors: 11,
     mentees_on_waiting_list: 12,
     unregistered_mentees: 13,
-    unregistered_mentors: 14
+    unregistered_mentors: 14,
```
Comment (DMips) [inline app/models/email_template.rb]: please delete comma

File: app/models/email_template.rb (hunk 1 above)
Comment (DMips) [inline app/models/email_template.rb]: @pg20 this email will be send when mentor/mentee was reported as missing

File: db/schema.rb [hunk 3]
```diff
@@ -161,8 +161,8 @@
     t.boolean  "is_missing"
     t.string   "program_country"
     t.string   "timezone"
-    t.integer  "edition_id"
```
Comment (DMips) [inline db/schema.rb]: if there's no change to schema why commit it?

File: lib/tasks/issue_emails.rake [hunk 4]
```diff
@@ -0,0 +1,19 @@
+namespace :issue_emails do
+  desc "send emails to missing mentors"
+  task missing_mentors: :environment do
+     email_template = EmailTemplate.find_by(recipients: 4)
```
Comment (DMips) [inline lib/tasks/issue_emails.rake]: please delete this file as those email will be sent when user is reported as missing

File: db/seeds.rb [hunk 5]
```diff
@@ -71,3 +71,16 @@
...
 )
+
+EmailTemplate.create(
+  subject: "[Learn IT, Girl] Reported as missing. Action required",
+  recipients: 4,
+  body: "Dear %{first_name} % {last_name}\r\nYour mentee has reported you as missing in “Learn IT, Girl” as she has been unable to get in touch with you. Mentors are key part of “Learn IT, Girl”, as they help make the dream of learning to code come true for their mentee. For this dream to become true, communication is essential.\r\n\r\nIf there is any problem or issue causing this, please, let us know. We will do our best to help and remove any roadblocks! In order for your status not be marked as “abandoned” you should contact your mentee and us in the next two weeks.\r\n\r\nPlease let us know if you have any questions, problems or doubts. We will be happy to help!\r\n\r\nBest regards,\r\nThe “Learn IT, Girl” team\r\n<a href=\"http://www.learnitgirl.com\" target=\"_blank\">\"Learn IT, Girl\"</a> | <a href=\"https://twitter.com/learnitgirl\" target=\"_blank\">Follow us on Twitter </a>| <a href=\"https://www.facebook.com/learnitgirl\" target=\"_blank\">Like us on Facebook </a>| <a href=\"https://learnitgirl.wordpress.com\" target=\"_blank\">Read our blog</a>"
```
Comment (DMips) [inline db/seeds.rb]: can you get rid of space between last name and %? same in next template

File: db/schema.rb (hunk 3 above)
Comment (pg20) [inline db/schema.rb]: I didn't change anything. File must have changed in master :/

Comment (DMips) [inline db/schema.rb]: you run `rake db:migrate` and didn't checkout db/schema so even though you didn't add any migration db/schema was changed in your branch ;)

**Comment by DMips** [comment]
@pg20 looks good - i'll deploy it later

**Comment by pg20** [comment]
@DMips  before merging remind me to update db:seed, It has different timing for mentor and mentee to reply back to us. It was changed later in templates :)

**Comment by pg20** [comment]
@DMips  I have tested it. You can merge now. Just to recall: This pr send a mail to missing mentor when mentee  presses "report missing mentor" button from website and viceversa.

File: app/controllers/mentee_profiles_controller.rb [hunk 6]
```diff
@@ -6,7 +6,8 @@ def missing_mentor
 
     if @mentor.update_attributes(is_missing: true, missing_since: DateTime.now)
       MissingPersonsMailer.missing_mentor(@user).deliver_now
-      redirect_to dashboard_mentee_profiles_path, notice: 'Organisers have been notified about the missing mentor.'
+      send_email_to_missing_mentor
+      redirect_to dashboard_mentee_profiles_path, notice: 'Organsiers Have been notified about the missing mentor'
```
Comment (DMips) [inline app/controllers/mentee_profiles_controller.rb]: have instead of Have ;)

Comment (pg20) [inline app/controllers/mentee_profiles_controller.rb]: done @DMips .U can merge now (y)

**Pull request merged**